tdrs-backend/tdpservice/parsers/test/test_parse.py

```diff
@@ -888,3 +889,47 @@ def test_parse_tanf_section4_file(tanf_section4_file):
 
     assert first.FAMILIES_MONTH == 274
     assert sixth.FAMILIES_MONTH == 499
+
+@pytest.fixture
+def tribal_section_1_file(stt_user, stt):
+    """Fixture for ADS.E2J.FTP4.TS06."""
+    return util.create_test_datafile('ADS.E2J.FTP1.TS142', stt_user, stt, "Tribal Active Case Data")
+
+@pytest.mark.django_db()
+def test_parse_tribal_section_1_file(tribal_section_1_file):
+    """Test parsing Tribal TANF Section 1 submission."""
+    parse.parse_datafile(tribal_section_1_file)
+
+    assert Tribal_TANF_T1.objects.all().count() == 121
+    assert Tribal_TANF_T2.objects.all().count() == 199
+    assert Tribal_TANF_T3.objects.all().count() == 213
```

Comment:
Didn't we posit that hardcoding numbers like this leads to a maintenance tax for future changes? IIRC we didn't want to do so. @raftmsohani

Reply:
I think we should parse a file that has only one T1, T2 and T3 and assert some of the parsed values. Parsing a large file doesn't help in this case

Reply:
I thought we only wanted that for the parser errors since there is not an easy way to determine the exact number of errors in each file? We can always figure out how many records of each type are in a file with a quick grep.